docs/app/Components/ComponentDoc/ComponentExample/ComponentExample.js

```diff
@@ -441,7 +441,7 @@ class ComponentExample extends Component {
     }
 
     return (
-      <Visibility once={false} onTopPassed={this.handlePass} onTopPassedReverse={this.handlePass}>
```

Comment:
The right menu no longer updates as the user scrolls.  These handlers enabled that functionality.

Reply:
Oh! I didn't check that. Thank you for pointing it out. I'll look into it. 
And, all your comments make sense to me, I'll fix them.